backend/sql/init.sql

```diff
@@ -0,0 +1,172 @@
+-- for each concentration. Usually there are multiple repetitions,
+-- so more than 1 value for each concentration.
+create table if not exists experiment
+(
+    experiment_id       serial  not null primary key,
+    -- File from which this experiment was taken.
+    -- Project, methodology and some other data are inherited from that file.
+    experiment_file_id  int     not null,
+    compound_id         int     not null,
+    target_id           int     not null,
+
+    constraint has_compound
+        foreign key (compound_id) references compound (compound_id)
+            -- Compound id is immutable
+            on update no action
+            -- Deletion is currently not supported for compounds
+            on delete no action,
+
+    constraint has_target
+        foreign key (target_id) references target (target_id)
+            -- Target id is immutable
+            on update no action
+            -- Deletion is currently not supported for targets
+            on delete no action
+);
+
+create table if not exists measurement
+(
+    measurement_id  serial          not null primary key,
+    experiment_id   int             not null,
+    concentration   double precision not null,
+    signal          double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete no action
+            on update no action
+);
+
+create table if not exists analysis_method
+(
+    analysis_method_id serial not null primary key,
+    description        text   null,
+    -- Analysis-specific parameters
+    parameters         json   null
+);
+
+-- Subexperiment is an experiment with disabled points.
+-- An experiment is a subexperiment of itself with 0 disabled points.
+-- We define analysis for subexperiments.
+-- For each subexperiment we store information about applied analysis method
+-- and analysis outcome (result).
+-- If we use multiple analysis methods for one set of data, there will be
+-- multiple subexperiments.
+-- Additionally, we specify whether a subexperiment is suspicious
+-- (something looks wrong in its data).
+create table if not exists subexperiment
+(
+    subexperiment_id   serial not null primary key,
+    analysis_method_id int    not null,
+    experiment_id      int    not null,
+    is_suspicious      bool   not null,
+    result             json   not null,
+
+    constraint determined_by_analysis_method
+        foreign key (analysis_method_id) references analysis_method (analysis_method_id)
+            on delete no action
+            on update no action,
+
+    constraint such_experiment_exists
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete no action
+            on update no action
+
+    -- TODO do we want to make sure that there is at least one measurement for (experiment_id, compound_id) in
+    -- measurement table?
```

Comment:
Currently removal will be defined only for test methodologies, so it won't be possible to remove experiments. It's an old comment and I didn't want to touch it because I don't know if we want such constraint :)

I mean, I can add a constraint checking that each experiment and sub-experiment has at least one point, but I am not sure it will be useful. I think it will be clearer when we implement relevant functionality, that's why I keep it here for now.